src/test/java/org/broadinstitute/hellbender/tools/walkers/annotator/FisherStrandUnitTest.java

```diff
@@ -216,11 +216,12 @@ public void testUsingLikelihoods_Raw() {
         Assert.assertNotNull(annotatedMapRaw, vc.toString());
         final String actualStringRaw = (String) annotatedMapRaw.get(key);
```

Comment:
If you're going to replace key below you might as well do it here too and get rid of key.

Reply:
done